de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ArchiveIncomingProjectNegotiation.java

```diff
@@ -94,10 +78,10 @@ protected void transfer(IProgressMonitor monitor,
             final String projectID = entry.getKey();
             final IProject project = entry.getValue();
             /*
-             * TODO Move enable (and disable) queuing responsibility to
-             * SarosSession, since the second call relies on the first one, and
-             * the first one is never done without the second. (see also TODO in
-             * {@link cleanup}).
+             * TODO Queuing responsibility should be moved to Project
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=164&issues=AWTNsbFXi6fIhhPMrSiB&open=AWTNsbFXi6fIhhPMrSiB)

[ISSUE_KEY]: AWTNsbFXi6fIhhPMrSiB